2015-04-28 09:59:27,140 [ERROR]=========aaabbb
    System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 09:59:28,357 [ERROR]=========aaabbb
    System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 09:59:28,525 [ERROR]=========aaabbb
    System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 09:59:53,045 [ERROR]=========aaabbb
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 09:59:53,254 [ERROR]=========aaabbb
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 09:59:53,438 [ERROR]=========aaabbb
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 10:00:55,104 [ERROR]=========System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
   2015-04-28 10:02:47,017 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 10:04:21,750 [ERROR]=========System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
   2015-04-28 10:04:58,946 [ERROR]=========
   2015-04-28 10:05:01,496 [ERROR]=========
   2015-04-28 10:05:33,217 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 242015-04-28 10:05:33,404 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 242015-04-28 10:05:34,523 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 242015-04-28 10:06:12,049 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 10:06:12,259 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 10:06:12,432 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 10:07:19,332 [ERROR]=========
   System.Exception: 出错
2015-04-28 10:18:48,690 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 10:31:28,974 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 10:31:52,944 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 10:32:01,753 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 10:32:03,652 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 10:32:03,824 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 10:32:17,966 [ERROR]=========
   System.DivideByZeroException: 尝试除以零。
   在 EasyOa.Web.Controllers.HomeController.Index() 位置 d:\02_project_test\EasyOa.Web\EasyOa.Web\Controllers\HomeController.cs:行号 24
2015-04-28 14:28:22,715 [ERROR]=========
   System.Net.Mail.SmtpException: 发送邮件失败。 ---> System.Net.WebException: 未能解析此远程名称: 'smtp.exmail.qq.com'
   在 System.Net.ServicePoint.GetConnection(PooledStream PooledStream, Object owner, Boolean async, IPAddress& address, Socket& abortSocket, Socket& abortSocket6)
   在 System.Net.PooledStream.Activate(Object owningObject, Boolean async, GeneralAsyncDelegate asyncCallback)
   在 System.Net.PooledStream.Activate(Object owningObject, GeneralAsyncDelegate asyncCallback)
   在 System.Net.ConnectionPool.GetConnection(Object owningObject, GeneralAsyncDelegate asyncCallback, Int32 creationTimeout)
   在 System.Net.Mail.SmtpConnection.GetConnection(ServicePoint servicePoint)
   在 System.Net.Mail.SmtpTransport.GetConnection(ServicePoint servicePoint)
   在 System.Net.Mail.SmtpClient.GetConnection()
   在 System.Net.Mail.SmtpClient.Send(MailMessage message)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.Mail.SmtpClient.Send(MailMessage message)
   在 EasyOa.Common.MailHelper.Send(String subject, String message, String[] to) 位置 d:\02_project_test\EasyOa.Web\EasyOa.Common\Helpers\MailHelper.cs:行号 43
2015-04-28 15:31:20,713 [ERROR]=========
   System.Net.Mail.SmtpException: 操作已超时。
   在 System.Net.Mail.SmtpClient.Send(MailMessage message)
   在 EasyOa.Common.MailHelper.Send(String subject, String message, Boolean isHtml, List`1 fileCollection, String[] to) 位置 d:\02_project_test\EasyOa.Web\EasyOa.Common\Helpers\MailHelper.cs:行号 70
2015-04-28 15:38:49,888 [ERROR]=========
   System.Net.Mail.SmtpException: 操作已超时。
   在 System.Net.Mail.SmtpClient.Send(MailMessage message)
   在 EasyOa.Common.MailHelper.Send(String subject, String message, Boolean isHtml, List`1 fileCollection, String[] to) 位置 d:\02_project_test\EasyOa.Web\EasyOa.Common\Helpers\MailHelper.cs:行号 81
2015-04-28 15:53:14,181 [ERROR]=========
   System.Net.Mail.SmtpException: 发送邮件失败。 ---> System.InvalidOperationException: 现在无法开始异步操作。异步操作只能在异步处理程序或模块中开始，或在页生存期中的特定事件过程中开始。如果此异常在执行 Page 时发生，请确保 Page 标记为 <%@ Page Async="true" %>。
   在 System.Web.AspNetSynchronizationContext.OperationStarted()
   在 System.ComponentModel.AsyncOperation.CreateOperation(Object userSuppliedState, SynchronizationContext syncContext)
   在 System.Net.Mail.SmtpClient.SendAsync(MailMessage message, Object userToken)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.Mail.SmtpClient.SendAsync(MailMessage message, Object userToken)
   在 EasyOa.Common.MailHelper.SendAsync(String subject, String message, Boolean isHtml, List`1 fileCollection, String[] to) 位置 d:\02_project_test\EasyOa.Web\EasyOa.Common\Helpers\MailHelper.cs:行号 117
2015-04-28 15:56:47,202 [ERROR]=========
   System.Net.Mail.SmtpException: 发送邮件失败。 ---> System.InvalidOperationException: 现在无法开始异步操作。异步操作只能在异步处理程序或模块中开始，或在页生存期中的特定事件过程中开始。如果此异常在执行 Page 时发生，请确保 Page 标记为 <%@ Page Async="true" %>。
   在 System.Web.AspNetSynchronizationContext.OperationStarted()
   在 System.ComponentModel.AsyncOperation.CreateOperation(Object userSuppliedState, SynchronizationContext syncContext)
   在 System.Net.Mail.SmtpClient.SendAsync(MailMessage message, Object userToken)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.Mail.SmtpClient.SendAsync(MailMessage message, Object userToken)
   在 EasyOa.Common.MailHelper.SendAsync(String subject, String message, Boolean isHtml, List`1 fileCollection, String[] to) 位置 d:\02_project_test\EasyOa.Web\EasyOa.Common\Helpers\MailHelper.cs:行号 117
2015-04-28 17:07:26,714 [ERROR]=========
   System.Net.Mail.SmtpFailedRecipientException: 邮箱不可用。 服务器响应为: Mailbox not found or access denied
   在 System.Net.Mail.SmtpTransport.SendMail(MailAddress sender, MailAddressCollection recipients, String deliveryNotify, Boolean allowUnicode, SmtpFailedRecipientException& exception)
   在 System.Net.Mail.SmtpClient.Send(MailMessage message)
   在 EasyOa.Common.MailHelper.Send(String subject, String message, Boolean isHtml, List`1 fileCollection, String[] to) 位置 d:\02_project_test\EasyOa.Web\EasyOa.Common\Helpers\MailHelper.cs:行号 92
2015-04-28 17:19:19,334 [ERROR]=========
   
2015-04-28 17:19:44,833 [ERROR]=========
   System.Net.Mail.SmtpFailedRecipientException: 邮箱不可用。 服务器响应为: Mailbox not found or access denied
   在 System.Net.Mail.SmtpTransport.SendMail(MailAddress sender, MailAddressCollection recipients, String deliveryNotify, Boolean allowUnicode, SmtpFailedRecipientException& exception)
   在 System.Net.Mail.SmtpClient.Send(MailMessage message)
   在 EasyOa.Common.MailHelper.Send(String subject, String message, Boolean isHtml, List`1 fileCollection, String[] to) 位置 d:\02_project_test\EasyOa.Web\EasyOa.Common\Helpers\MailHelper.cs:行号 92  

2015-04-28 17:19:47,336 [ERROR]=========
   System.Net.Mail.SmtpFailedRecipientException: 邮箱不可用。 服务器响应为: Mailbox not found or access denied
   在 System.Net.Mail.SmtpTransport.SendMail(MailAddress sender, MailAddressCollection recipients, String deliveryNotify, Boolean allowUnicode, SmtpFailedRecipientException& exception)
   在 System.Net.Mail.SmtpClient.Send(MailMessage message)
   在 EasyOa.Common.MailHelper.Send(String subject, String message, Boolean isHtml, List`1 fileCollection, String[] to) 位置 d:\02_project_test\EasyOa.Web\EasyOa.Common\Helpers\MailHelper.cs:行号 92

